### PR TITLE
restructure github silk

### PR DESCRIPTION
**Pull request by jenniexie**
This PR is clean up Github Silk repo:
1. changed src to silk
2. changed setup.py and README.md

### CODE REVIEW

File: silk/tests/silk_run.py [hunk 1]
```diff
@@ -198,5 +198,5 @@ def run(self):
 
 
 if __name__ == "__main__":
-
+    sys.argv = ['tests/silk_run.py', '-v2', '-c', CONFIG_PATH+'hwconfig.ini', '-d', RESULT_LOG_PATH,  'ot_test_form_network.py']
```
Comment (qilinkang) [inline silk/tests/silk_run.py]: Please remove or comment out this line which is for  debug run

File: silk/unit_tests/silk_run_test.py [hunk 2]
```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from src.tests import silk_run
+from silk.tests import silk_run
 import os
 import datetime
 
-RESULT_LOG_PATH = '/opt/openthread_test/results/'+'silk_run_0502_A/'
+RESULT_LOG_PATH = '/opt/openthread_test/results/'+'silk_run/'
```
Comment (qilinkang) [inline silk/unit_tests/silk_run_test.py]: Suggest to append the current date to 'silk_run_' so all tests in the same day will be in one folder for easy access.

File: silk/unit_tests/silk_run_test.py [hunk 3]
```diff
@@ -25,7 +25,7 @@
 
 run_log_path = RESULT_LOG_PATH + 'test_run_on_' + timestamp + '/'
 
-argv = ['tests/silk_run.py', '-v2', '-c', CONFIG_PATH+'hwconfig.ini', '-d', run_log_path, 'ot_test*.py']
+argv = ['tests/silk_run.py', '-v2', '-c', CONFIG_PATH+'hwconfig.ini', '-d', run_log_path, 'ot_test_leader_loss.py']
```
Comment (qilinkang) [inline silk/unit_tests/silk_run_test.py]: Better to keep the original code to run  all test suites.

**Review by qilinkang — CHANGES_REQUESTED** [review]

File: silk/tests/silk_run.py (hunk 1 above)
Comment (jenniexie) [reply silk/tests/silk_run.py]: I did comment, so we have one line to keep the example, and use it for debugging purpose for future

File: silk/unit_tests/silk_run_test.py (hunk 2 above)
Comment (jenniexie) [reply silk/unit_tests/silk_run_test.py]: will fix it

File: silk/unit_tests/silk_run_test.py (hunk 3 above)
Comment (jenniexie) [reply silk/unit_tests/silk_run_test.py]: will fix it

**Comment by jenniexie** [comment]
I have fixed all comments, please help to review again @qilinkang

**Review by qilinkang — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**